imap_processing/tests/lo/test_anc/imap_lo_l1c-sweeptable_20250415_v001.csv

```diff
@@ -0,0 +1,3 @@
+YYYYDDD,GoodTime_strt,GoodTime_end
```

Comment:
Is it a typo with `strt` missing the "a" or that is expected?

The table header says "DDD" but it looks like you are putting in "MMDD" in the values below.

Reply:
That's how it's spelled in the sample ancillary file that Lo sent. I'll fix the date issue. I got a new file from them that has some changes so I'll add those changes to this PR